nemo/collections/llm/gpt/model/base.py

```diff
@@ -310,6 +311,15 @@ def validation_step(self, batch, batch_idx=None) -> torch.Tensor:
 
         return self.forward_step(batch)
 
+    def get_inference_wrapper(self, inference_wrapper_config) -> torch.Tensor:
+        # get mcore, unwrapped model
+        mcore_model = self.module
```

Comment:
I think it would be safer to have a while loop that checks `isinstance(module, MCoreGPTModel)`

Reply:
Sure. I can fix this.